packages/esm-hiv-app/src/forms/care-and-treatment/mental-health/1.0.json

```diff
@@ -73,6 +93,17 @@
 									}
 								]
 							},
+							"behaviours": [
+								{
+									"intent": "*",
+									"required": "true",
+									"unspecified": "true",
+									"hide": {
+										"hideWhenExpression": "false"
+									  }
+								}
+								
+							],
 							"id": "pHQ2_1"
```

Comment:
@kajambiya you should maintain a consistent naming convention. Camel case is what we have been using mostly.